wapitiCore/main/wapiti.py

```diff
@@ -648,16 +648,25 @@ def have_attacks_started(self) -> bool:
         return self.persister.have_attacks_started()
 
 
-def is_valid_endpoint(url):
+def set_url_path(url):
+    """Set the url path if its not defined"""
+    return url if urlparse(url).path else url + '/'
+
+
+def is_valid_url(url_type, url):
+    """Verify if the url provided has the right format"""
     try:
         parts = urlparse(url)
     except ValueError:
+        print('ValueError')
         return False
     else:
         if parts.params or parts.query or parts.fragment:
+            print(_("Error: {} must not contain params, query or fragment!").format(url_type))
```

Comment:
This is not good for the URL given to wapiti using -u. You can provide an URL containing parameters if you are using the "url" scope

Reply:
I reverted the changes for the main *url*